client/client.go

```diff
@@ -369,6 +372,24 @@ const (
 	WorkflowIDReusePolicyRejectDuplicate WorkflowIDReusePolicy = internal.WorkflowIDReusePolicyRejectDuplicate
 )
 
+// ParentClosePolicyAbandon means not doing anything on the child workflow
+func ParentClosePolicyAbandon() *internal.ParentClosePolicy {
+	p := internal.ParentClosePolicyAbandon
+	return &p
+}
+
+// ParentClosePolicyRequestCancel means requesting cancellation on the child workflow
+func ParentClosePolicyRequestCancel() *internal.ParentClosePolicy {
+	p := internal.ParentClosePolicyRequestCancel
+	return &p
+}
+
+// ParentClosePolicyTerminate means terminating the child workflow
+func ParentClosePolicyTerminate() *internal.ParentClosePolicy {
+	p := internal.ParentClosePolicyTerminate
+	return &p
+}
+
```

Comment:
Why is this so different from IDReusePolicy? Why the function instead of a constant?